### PR TITLE
Just add  Version 0.14 description for README.md

### PR DESCRIPTION
**Pull request by deepinfan**
There is some fix for 0.13 by @Lasoloz Contributor 9 months ago.

Just  add  Version 0.14 description for README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ### py_sg
 
+Version 0.14
+
+Add missingy macro for newer Python 3 versions
```
Comment (Lasoloz) [inline README.md]:
```suggestion
Add missing macro for newer Python 3 versions
```

File: README.md [hunk 2]
```diff
@@ -20,6 +24,8 @@ sudo python3 -m pip install <path/to/setup.py>
 
 ### Changelog
 
+Version 0.14 - Missing PY_SSIZE_T_CLEAN macro added, fix for newer Python 3 versions
```
Comment (Lasoloz) [inline README.md]: I forgot to specify the Python 3 version in my PR back then. Maybe the exact Python 3 version could be specified (which makes the macro mandatory)?

**Review by Lasoloz — COMMENTED** [review]
Thanks, @deepinfan, I didn't notice that I should've updated the README.

File: README.md (hunk 2 above)
Comment (tvladyslav) [reply README.md]: Good point, let me specify exact version - 3.10

**Pull request merged**